public/styles.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+    font-family: "Source Sans Pro";
+    src: url(assets/source-sans-pro-v9-latin-regular.woff);
+}
+
+html {
+    background-color: #F3F5F6;
+}
+
+body {
+    margin: 0;
+}
+
+.header {
+    display: flex;
+    position: relative;
+    width: 100%;
+    height: 60px;
+    justify-content: space-between;
+    align-items: center;
+    background-color: #F9FAFA;
```

Comment:
We should be using variables for colors. Please look at the `CSS variables` from `pi-ui`